src/commands/protocols/common/submit-proofs-command.js

```diff
@@ -161,10 +161,24 @@ class SubmitProofsCommand extends Command {
                 txGasPrice,
             );
         });
-
+        const sendSubmitProofsTransactionOperationId = this.operationIdService.generateId();
         let txSuccess;
         try {
+            this.operationIdService.emitChangeEvent(
+                OPERATION_ID_STATUS.COMMIT_PROOF.SUBMIT_PROOFS_SEND_TX_START,
+                sendSubmitProofsTransactionOperationId,
+                blockchain,
+                agreementId,
+                epoch,
+            );
             txSuccess = await transactionCompletePromise;
+            this.operationIdService.emitChangeEvent(
```

Comment:
Do we want to emit END event if tx fails because proof was already submitted or maybe we should use some other status?

Reply:
Error event will be emitted with error message

Reply:
But we're not going into catch clause for errors that are "Already submitted"